client/packages/system/src/Patient/Service/Service.tsx

```diff
@@ -1,11 +1,17 @@
 import React, { FC } from 'react';
-import { Routes, Route } from '@openmsupply-client/common';
+import { Routes, Route, RouteBuilder } from '@openmsupply-client/common';
 import { PatientListView } from '../ListView';
+import { PatientDetailView } from '../DetailView';
+
+const singlePatientRoute = RouteBuilder.create(':patientId')
+  .addPart(':docType')
+  .build();
 
 export const Service: FC = () => {
   return (
     <Routes>
       <Route path={''} element={<PatientListView />} />
+      <Route path={singlePatientRoute} element={<PatientDetailView />} />
```

Comment:
I wasn't sure why there is a `Patient` in the URL when I'm looking at the patient.

Also the `patient2` in the breadcrumb takes me to a blank page:

<img width="574" alt="Screen Shot 2022-06-28 at 11 15 47 AM" src="https://user-images.githubusercontent.com/9192912/176054357-daeefe00-6bbd-405c-a7f5-1545873b5601.png">
 
Partly solved by #182 I think

Reply:
Okay, have removed the "type" component as discussed. Can add to this as we need it.

2e2b5c753